Tools/PIDML/qaPidML.cxx

```diff
@@ -321,7 +321,7 @@ struct pidml {
   template <std::size_t i, typename T>
   void pidML(const T& track, const int pdgCodeMC)
   {
-    int pidLogits[3] = {model211.applyModel(track), model2212.applyModel(track), model321.applyModel(track)};
+    float pidLogits[3] = {int(model211.applyModel(track)), model2212.applyModel(track), model321.applyModel(track)};
```

Comment:
You require an int array in pidParticle(), this causes problems.
Could you simply change the pidParticle signature to `const float pidLogits`?
And don't make ints out of elements of a float array, then you get conversion errors.